src/coreimage.cs

```diff
@@ -1819,27 +1819,26 @@ interface CIWarpKernel {
 
 	[iOS (9,0)]
 	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor] // does not work in iOS 11 beta 4
 	interface CIImageAccumulator {
+#if XAMCORE_4_0
+		[Obsolete ("The default initializer does not work in recent iOS version (11b4).")]
+		IntPtr Constructor ();
+#endif
+
 		[Static]
 		[Export ("imageAccumulatorWithExtent:format:")]
-#if !MONOMAC
-		[Internal]
-#endif
 		CIImageAccumulator FromRectangle (CGRect rect, int /* CIFormat = int */ ciImageFormat);
 
 		[iOS (9,0)]
-		[Static, Internal]
+		[Static]
```

Comment:
I have not _yet_ gone to the root of this :)
It's not clear why those methods were half hidden (inconsistent) in XM and not exposed at all (unusable) in XI.